src/app/pages/apps/components/available-apps/app-available-info-card/app-available-info-card.component.ts

```diff
@@ -14,16 +17,31 @@ import { AvailableApp } from 'app/interfaces/available-app.interface';
 export class AppAvailableInfoCardComponent implements OnChanges {
   @Input() isLoading$: Observable<boolean>;
   @Input() app: AvailableApp;
+  sources: string[] = [];
+  loadingSources = true;
   relativeDate = '';
 
   constructor(
     private cdr: ChangeDetectorRef,
+    private applicationService: ApplicationsService,
   ) { }
 
   ngOnChanges(): void {
     if (!this.app) {
       return;
     }
+    this.loadingSources = true;
+    this.applicationService.getCatalogItem(this.app.name, this.app.catalog, this.app.train)
```

Comment:
Let's make a ticket for middleware to add this field to available apps response.